infra/main.bicep

```diff
@@ -190,7 +190,7 @@ module openAiRoleUser 'core/security/role.bicep' = {
   params: {
     principalId: principalId
     roleDefinitionId: '5e0bd9bd-7b93-4f28-af87-19fc36ad61bd'
-    principalType: 'User'
+    principalType: 'ServicePrincipal'
```

Comment:
These roles are for the local user, please do not change to ServicePrincipal

Reply:
done